proteus/Transport.py

```diff
@@ -3555,9 +3559,11 @@ def calculateExteriorElementBoundaryCoefficients(self):
                 if self.coefficients.advection.has_key(ci):
                     self.ebqe[('advectiveFlux',ci)][t[0],t[1]] = g(self.ebqe[('x')][t[0],t[1]],self.timeIntegration.t)
                     for cj in self.coefficients.advection[ci].keys():
+                        #
```

Comment:
I would try to minimize this kind of touching of the source that doesn't really introduce a change when it's in the context of a feature request. As we converge on formatting styles we can do those kinds of reviews separately.